apps/ensindexer/src/lib/plugin-helpers.ts

```diff
@@ -194,12 +194,42 @@ export function networkConfigForContract<CONTRACT_CONFIG extends ContractConfig>
 ) {
   return {
     [chain.id.toString()]: {
-      ...contractConfig,
-      ...constrainContractBlockrange(contractConfig.startBlock),
+      address: contractConfig.address, // provide per-network address if available
```

Comment:
note: this explicitly only passes `address`, and the blockrange (`startBlock`, `endBlock`) to the per-network contract config overrides, to avoid bloating the ponder config with unnecessary values (like `abi`)